frontend/public/components/factory/table.tsx

```diff
@@ -296,7 +300,16 @@ const VirtualBody: React.FC<VirtualBodyProps> = (props) => {
         parent={parent}
         rowIndex={index}
       >
-        {row}
+        <TableRow
+          id={getRowId?.(rowArgs.obj) ?? key}
+          index={index}
+          trKey={key}
+          style={style}
+          title={getRowTitle?.(rowArgs.obj)}
+          className={getRowClassName?.(rowArgs.obj)}
+        >
+          <RowMemo {...rowArgs} Row={Row} />
+        </TableRow>
```

Comment:
This is the important change.
Scrolling causes `style` prop to change so I placed the `TableRow` component (the only one using `style`) and put the rest behind `PureComponent`.
Scrolling now causes only `TableRow` to rerender, the rest is kept as-is.

Since I moved `TableRow` here I had to update all table rows.

Reply:
Looking at the React profiler, when scrolling on pods page, rerendering takes about 2ms instead of 20ms before this change. That is however hightly dependant on how fast are we scrolling, how many new items are rendered, table content etc.
Anyways..the items which are already rendered are not being re-rendered only due to scrolling.